cinch/roles/nginx/tasks/ssl-setup.yml

```diff
@@ -0,0 +1,49 @@
+- name: copy over ssl key
+  copy:
```

Comment:
hmm, good question. I tried to modify the existing role as little as possible, so I'm not exactly sure... 
It isn't really used anyway, since the SSL cert which is really used gets copied in roles/jenkins_master/tasks/https.yml
I'll try to simplify the role as much as possible to remove all this unnecessary stuff since we agreed to incorporate external roles in later phases...